src/TRIANGLE.HOWTO

```diff
@@ -15,12 +15,12 @@ If you later need to change the behavior w.r.t. triangulation,
 follow these guidelines:
 
 In makegmt.macro there are two parameters that can be defined:
-TRIANGLE_D and TRIANGLE_O.  By default they are commented out.
+TRIANGLE_D and TRIANGLE_O.  By default these lines are commented out.
 To use Shewchuk's [1996] routine, remove the # from the
 beginning of the lines and then 'make clean' and 'make install'.
-To revert to the default GMT usage of Watson's [1982] algorithm,
+To revert to using Watson's [1982] algorithm,
 simply place a # in the first column at these two lines.
-Then, say 'make clean' and 'make install' again.
+Then, run 'make clean' and 'make install' again.
```

Comment:
I need a bit of help rewording these lines. I don't see `makegmt.macro` anywhere (the `git blame` on this file dates back to Dec 2000!!). I do see a `TRIANGLE_D` here:

https://github.com/GenericMappingTools/gmt/blob/434ed1848a1e8886931694b2de4c352e4c6cddae/src/gmt_init.c#L6580-L6585

but no mention of `TRIANGLE_O`.

Reply:
My best guess is that these macros are in reference to the pre-CMake days. I think that this paragraph and the preceding to sentences should instead make these points:
1. Use `gmt set GMT_TRIANGULATE` to control which algorithm is used on the fly.
2. Use `set (LICENSE_RESTRICTED GPL)` in `cmake/ConfigUserAdvanced.cmake` to disable Shewchuk's routine entirely.

Reply:
Yes, pre-CMake language.